agent/unit-test/containeroptionparser_unittest.cpp

```diff
@@ -63,6 +63,40 @@ TEST_F(ContainerOptionParserTest, parseConfigNice2) {
     ASSERT_FALSE(m_options->enableWriteBuffer());
 }
 
+TEST_F(ContainerOptionParserTest, parseConfigNice3) {
```

Comment:
I would prefer if the names of the test cases were something a little more descriptive. Also for the first and second of the "Nice" test cases. Just a preference. It makes it so much easier to understand what's going on in the test case without reading the full log.